web/src/cypress/integration/scheduleCalendar.ts

```diff
@@ -87,7 +87,7 @@ function testCalendar(screen: ScreenFormat): void {
   })
 
   it(`should view a shift's tooltip`, () => {
-    cy.get('div').contains(rot.users[0].name).trigger('mouseover')
```

Comment:
We should keep this and add a second assertion unless we're removing the ability to hover.

Reply:
That's correct, this PR would remove hovering so that a tooltip appears when clicked or focused (screen readers, tab, etc)

Reply:
We should discuss this first before making UX changes -- let's get the release blockers out of the way first

Reply:
I feel if we are making UX changes to error handling this late, it's okay to get something like this working better for our users too

Reply:
I definitely agree that we should get release blockers out of the way first — David pointed out this should probably go into master instead so I'll get the base branch updated to give us more space to chat about this 👍